packages/core-transaction-pool/src/errors.ts

```diff
@@ -0,0 +1,95 @@
+import { Contracts } from "@arkecosystem/core-kernel";
+import { Interfaces, Utils } from "@arkecosystem/crypto";
+
+export class DuplicateError extends Error implements Contracts.TransactionPool.DuplicateError {
```

Comment:
Is there a specific reason for needing interfaces on error classes? They all simply implement the `constructor` method for all they need to know.

Reply:
No reason. I'm removing them.